src/lib/get-custom-properties-from-root.js

```diff
@@ -48,8 +48,8 @@ const rootSelectorRegExp = /^:root$/i;
 const customPropertyRegExp = /^--[A-z][\w-]*$/;
 
 // whether the node is an html or :root rule
-const isHtmlRule = node => node.type === 'rule' && htmlSelectorRegExp.test(node.selector) && Object(node.nodes).length;
-const isRootRule = node => node.type === 'rule' && rootSelectorRegExp.test(node.selector) && Object(node.nodes).length;
+const isHtmlRule = node => node.type === 'rule' && -1 !== node.selector.split(',').findIndex(item => htmlSelectorRegExp.test(item)) && Object(node.nodes).length;
+const isRootRule = node => node.type === 'rule' && -1 !== node.selector.split(',').findIndex(item => rootSelectorRegExp.test(item)) && Object(node.nodes).length;
```

Comment:
I think this function can be simplified as well.

```suggestion
const isRootRule = node => node.type === 'rule' && -1 !== node.selectors.some(item => rootSelectorRegExp.test(item)) && Object(node.nodes).length;